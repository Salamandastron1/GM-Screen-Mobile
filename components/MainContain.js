import React from 'react';
import { View } from 'react-native';
import PropTypes from 'prop-types';

const MainContain = ({ children }) => (
  <View style={styles.containerStyle}>
    {children}
  </View>
);

const styles = {
  containerStyle: {
    flex: 1,
    padding: 5,
    alignItems: 'center',
    backgroundColor: '#f4d8b7',
    justifyContent: 'space-around',
  },
};

MainContain.defaultProps = {
  children: {} || [],
};

MainContain.propTypes = {
  children: PropTypes.object || PropTypes.array,
};

export default MainContain;
