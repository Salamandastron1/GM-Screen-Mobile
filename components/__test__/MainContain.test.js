/* eslint-disable */

import React from 'react';
import { shallow } from 'enzyme';
import MainContain from '../MainContain';

it('should match the shapshot', () => {
  const tree = shallow(<MainContain />);
  expect(tree).toMatchSnapshot();
});