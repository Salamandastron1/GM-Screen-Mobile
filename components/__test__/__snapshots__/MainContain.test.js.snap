// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the shapshot 1`] = `
<Component
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#f4d8b7",
      "flex": 1,
      "justifyContent": "space-around",
      "padding": 5,
    }
  }
/>
`;
